app/views/topics/show.html.erb

```diff
@@ -7,19 +7,7 @@
 <div class="row">
   <div class="col-md-8">
     <p class="lead"><%= @topic.description %></p>
```

Comment:
Not sure about how to add the topic partial here. In the show view, we're only displaying one topic, as opposed to the entire index. Wouldn't the partial call up the whole index of topics?